cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -114,6 +114,12 @@ message CassandraNode {
     required bool seed = 9;
     optional int64 lastRepairTimestamp = 10;
     optional int64 lastCleanupTimestamp = 11;
+
+    optional NodeLocation location = 12;
+}
+message NodeLocation {
+    required string datacenter = 1;
+    required string rack = 2;
```

Comment:
Hm - for on-prem installs DC + rack should be injected as Mesos node properties.
And maybe it's not a bad idea to use one framework instance per DC - otherwise one single scheduler would be responsible for all nodes all over the world, which is probably not a good idea. We should then add some inter-scheduler mechanism to exchange seed nodes. But since it's not something that changes often, it's might be fine to do that with some tool called "human" once.